tools/gen-ssh.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/bash
+
+# Create SSH key
+SSH_DATA_DIR="$(mktemp -d)"
+SSH_KEY=${SSH_DATA_DIR}/id_rsa
+ssh-keygen -f "${SSH_KEY}" -N "" -q -t rsa
+
+# Change cloud-init/user-data ssh key
+key=" - $(cat "${SSH_KEY}".pub)"
+# Temporary, will copy user data from cloud-init once
+# go test are updated
+tee "${SSH_DATA_dir}"/user-data > /dev/null << EOF
```

Comment:
This is case sensitive, so should be `SSH_DATA_DIR`, not `SSH_DATA_dir` :)

Reply:
ups, thanks!